data/_ui/tooltips.txt

```diff
@@ -1092,23 +1092,242 @@ tip "This outfit is considered an atrocity."
 
 
 # Controls
-tip `Reverse`
-	`Turn opposite to the flight direction (or use reverse engines if you have them installed). Combine with <shift> key to stop your ship.`
+tip "Forward thrust"
+	`Accelerate your flagship forward.`
+
+tip "Turn left"
+	`Turn your flagship left.`
+
+tip "Turn right"
+	`Turn your flagship right.`
+
+tip "Reverse"
+	`Turn your flagship opposite its current direction of flight. If reverse thrusters are installed, accelerate backwards. With <shift>, stop your ship.`
+
+tip "Fire afterburner"
+	`Fire your flagship's afterburner, if one is installed.`
+
+tip "Auto steer"
+	`If the "Automatic aiming" setting is off, activate automatic aiming when this key is held.`
+
+tip "Land on planet / station"
+	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on, or attempts to land on the planet you are directly on top of if traveling at low speeds. If a planet is already selected, double tapping will cycle to the next planet.`
+
+tip "Initiate hyperspace jump"
+	`Initiate a hyperspace jump to follow your current travel plan. If you have no travel plan, initiates a jump nearest to your current facing angle. When held, commands your escorts to get ready to jump and only initiates the jump when released. With <shift>, your fleet will not jump until every ship in the fleet is ready for every system in the travel plan instead of needing to hold the key at each system.`
+
+tip "View star map"
+	`Open the star map panel.`
+
+tip "View player info"
+	`Open the player info panel.`
+
+tip "Select nearest hostile ship"
+	`Select the nearest hostile ship. With <shift>, select the nearest ship regardless of hostility.`
+
+tip "Select next ship"
+	`Select the next non-escort ship in the system, cycling through all ships. With <shift>, cycle through escorts.`
+
+tip "Talk to selected ship"
```

Comment:
Maybe this keybind could use a new name?
It is very exclusive to ships, calling it just hail might work

Reply:
I was thinking we could rename a few controls, but I wanted to keep this focused on adding tooltips and leave any renaming until later. (Another one I wanted to rename was "Target asteroids based on" to "Asteroid target priority".)
![image](https://github.com/endless-sky/endless-sky/assets/17688683/08cb194b-56bc-42d9-9e06-a5e770f5530e)